src/blockchain.rs

```diff
@@ -428,29 +428,42 @@ impl FromStr for Network {
     }
 }
 
-impl From<bitcoin::Network> for Network {
```

Comment:
Can we get both directions?

Reply:
Yes we actually can. I rework the commits to not erase the work from @LeoNero as much as I can, thus I have to move the impl from the  `blockchain` module into `bitcoin`. If this bothers you @LeoNero I can remove that part and you do another PR yourself. I should have catch that in your first PR but I didn't, sorry for that.

Reply:
Let's merge this to keep the momentum going. Leo's work is accurately preserved here and if there is something bothering him with the PR here, he can always submit a fresh PR to add more docs/test/functionality.

Reply:
Hi just saw that. There is no problem removing my implementation for things to work properly or for better code organization.